packages/ui/src/pages/SocialSignInCallback/index.tsx

```diff
@@ -1,9 +1,13 @@
+import { useParams } from 'react-router-dom';
+
 import useSocialSignInListener from '@/hooks/use-social-sign-in-listener';
 
 import SignIn from '../SignIn';
 
 const SocialSignInCallback = () => {
-  useSocialSignInListener();
+  const parameters = useParams<{ connector: string }>();
```

Comment:
The parameter is actually `connectorId`, isn't it?

Reply:
that is a legacy issue, as the params are named as `connector` in the route path. Create a separate issue to replace globally.